lib/src/treebuilder.dart

```diff
@@ -71,21 +71,21 @@ bool _nodesEqual(Element node1, Element node2) {
 
 /// Basic treebuilder implementation.
 class TreeBuilder {
-  final String defaultNamespace;
+  final String? defaultNamespace;
 
-  Document document;
+  var document = Document();
```

Comment:
Technically this first one is always going to get dropped on the floor, semantically `late` is probably the better option but might come with other overhead we don't want?

Or maybe make it const if that is supported?

Reply:
`late` sounds good to me.